app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(
+            self,
+            name: str,
+            age: int,
+    ) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people.update({self.name: self})
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict]) -> list[Person]:
+    person_list = [
+        Person(
+            person.get("name"),
+            person.get("age")
+        )
+        for person in people
+    ]
+    for num, person in enumerate(people):
+        for key, value in person.items():
```

Comment:
This loop is not needed
Just two ifs:
1. if person.get("wife") is not None
2. if person.get("husband") is not None

Reply:
should I add "if" into list comprehension? I think, it will cause error because of 4 arguments given instead of 3, or there is another possibility for it?